Documentation/bootstrapping-guidelines.md

```diff
@@ -0,0 +1,96 @@
+
+To build .NET SDK from source requires previously source-built .NET SDK and toolset which is also referred as bootstrapping .NET SDK and toolset. Check quick [instructions](https://github.com/dotnet/installer/blob/main/README.md#build-net-from-source-source-build) to build SDK using source-build. This document provides guidance for getting or building  bootstrapping .NET SDK and toolset in different scenarios (covering step 2 and step 3 of source-build instructions).
+
+## Building New .NET Version
```

Comment:
It would also be good to see some examples in the docs. An end-to-end example (with all specific commmands), for instance, of bootstrapping on a specific OS.

Reply:
Good suggestion. We are working to gather these examples from specific Linux distributions. Once received, I will update the document.